src/inventory/chest_inventory.rs

```diff
@@ -256,8 +192,4 @@ impl Inventory for ChestInventory {
     ) {
         self.init(renderer, ui_container, inventory_window);
     }
-
-    fn ty(&self) -> InventoryType {
```

Comment:
If we were to remove inventory type getters, how would you enable plugins (or parts of the client code) to distinguish between different inventories/inventory types?